### ISSUE TITLE
Limit all spawning

### ISSUE THREAD

**Issue by rom1504** [PrismarineJS/flying-squid#224]
Including eggs

**Comment by Creeplays** [comment]
May be better way is to optimize physics?)

**Comment by rom1504** [comment]
Yes, but that's harder to do. So I'll just limit spawning for now.

**Comment by 109C** [comment]
So essentially a mob cap?

**Comment by rom1504** [comment]
Yes, I've already did it there https://github.com/PrismarineJS/flying-squid/blob/master/src/lib/plugins/spawn.js#L83 , just didn't check for eggs.
Could directly be done in initEntity too

**Comment by 109C** [comment]
I think it would be better to be done in `initEntity`, as it also would be the cap for naturally spawned entities, etc.
